xbmc/PlayListPlayer.cpp

```diff
@@ -984,7 +985,18 @@ void PLAYLIST::CPlayListPlayer::OnApplicationMessage(KODI::MESSAGING::ThreadMess
             return;
           }
           if (MUSIC::IsAudio(*item) || IsVideo(*item))
+          {
+            // Check if sources are locked - to capture media played via json-rpc
```

Comment:
Not sure this method is the right place to handle json rpc as this class here is fairly generic and handles much more than json rpc. I would try to do this check somewhere in xbmc/interfaces/json-rpc

Reply:
Is that not a good thing? - any video should not be playable without a password check when video lock is on.

Reply:
Then at least your comment is wrong.

Reply:
Changed.